src/side-loaded/basic-sideload/sideloaded.run.ts

```diff
@@ -0,0 +1,148 @@
+import { equal } from 'node:assert';
+import {
+  Field,
+  Mina,
+  Bytes,
+  Crypto,
+  createEcdsa,
+  createForeignCurve,
+  Hash,
+} from 'o1js';
+
+import {
+  EcdsaProgram,
+  KeccakProgram,
+  CounterProgram,
+  generateEcdsaProof,
+  generateKeccakProof,
+  generateCounterProof,
+} from './sideloaded-zkprograms.js';
+
+import { MyFavoriteZkApp } from './verifier-zkapp.js';
+
+/**
+ * This example showcases the side-loaded proof verification functionality:
+ * 1. Deploys a verifier contract that can verify ECDSA and Keccak proofs
+ * 2. Generates and verifies an ECDSA signature proof
+ * 3. Generates and verifies a Keccak hash preimage proof
+ * 4. Updates contract state after successful verification
+ */
+
+// ---------------------------- Setup Local Blockchain ----------------------------
+const localChain = await Mina.LocalBlockchain({
+  proofsEnabled: false,
+  enforceTransactionLimits: false,
+});
+Mina.setActiveInstance(localChain);
+const fee = 1e8;
+
+// Setup test accounts
+const [deployer, zkAppPrivateKey, alice] = localChain.testAccounts;
+const zkApp = new MyFavoriteZkApp(zkAppPrivateKey.key.toPublicKey());
```

Comment:
nit: same comment about a more descriptive contract name to reduce cognitive over head for first time readers.